File: Basics_with_JavaScript/910 For Loop/10 For Loop Excercise/05. Salary.js
function vladi(input) {
  let tabsOpened = Number(input[0]);
  let salary = Number(input[1]);
  flag = false;

  for (let index = 2; index < input.length; index++) {
    flag = false;
    let site = input[index];

    switch (site) {
      case "Facebook":
        salary -= 150;
        break;
      case "Instagram":
        salary -= 100;
        break;
      case "Reddit":
        salary -= 50;
        break;
    }
    if (salary <= 0) {
      break;
    } else {
      flag = true;
    }
  }
  if (flag) {
    console.log(Math.floor(salary));
  } else {
    console.log("You have lost your salary.");
  }
}

vladi([
  "10",
  "750",
  "Facebook",
  "Dev.bg",
  "Instagram",
  "Facebook",
  "Reddit",
  "Facebook",
  "Facebook",
]);
vladi(["3", "500", "Github.com", "Stackoverflow.com", "softuni.bg"]);
vladi(["3", "500", "Facebook", "Stackoverflow.com", "softuni.bg"]);

File: Basics_with_Python/PythonBasicCourse/4_For_Loop/10_For_Loop___Exercise/05._Salary.py
n = int(input())
salary = float(input())
lost = False
for number in range(1, n + 1):
    site_url = input()
    if site_url == "Facebook":
        salary -= 150
    elif site_url == "Instagram":
        salary -= 100
    elif site_url == "Reddit":
        salary -= 50
    if salary <= 0:
        lost = True
if lost:
    print(f"You have lost your salary.")
else:
    print(f"{int(salary)}")
